src/GovUk.Education.ExploreEducationStatistics.Admin/Services/BootstrapUsersService.cs

```diff
@@ -12,12 +14,12 @@ public class BootstrapUsersService
     {
         private readonly IConfiguration _configuration;
         private readonly UsersAndRolesDbContext _usersAndRolesDbContext;
-        private readonly UsersAndRolesDbContext _contentDbContext;
+        private readonly ContentDbContext _contentDbContext;
 
         public BootstrapUsersService(
             IConfiguration configuration,
             UsersAndRolesDbContext usersAndRolesDbContext,
-            UsersAndRolesDbContext contentDbContext)
+            ContentDbContext contentDbContext)
```

Comment:
Not sure what was going on here previously - I guess it got injected twice and because both had `Users` it worked? Surprised EF allows things to be injected twice like this.

Reply:
Hehe yeah, it's a little confusing, but `Users` on the `UsersAndRolesDbContext` is referring to the `AspNetUsers` table, whereas in `ContentDbContext` it's referring to our "internal" Users table.